.circleci/config.yml

```diff
@@ -443,6 +443,18 @@ jobs:
                 paths:
                     - ~/.cache/yarn
                     - node_modules
+            - run:
+                command: tools/parse-release-notes parse --android --destination android/fastlane/metadata/android/de-DE/changelogs/default.txt --language de --production
```

Comment:
Should `--production` always be set in this section?

Reply:
Yes with the exception of release notes for github.